prereise/gather/griddata/hifld/data_access/load.py

```diff
@@ -22,12 +24,38 @@ def get_eia_form_860(path):
     return data.query("State in @abv2state")
 
 
-def get_epa_ampd(path, year=2019):
+def get_eia_epa_crosswalk(path):
+    """Read a CSV file mapping EIA plants IDs to EPA plant IDs, keep non-retired plants.
+
+    :param str path: path to file. Either local or URL.
+    :return: (*pandas.DataFrame*) -- filtered data frame.
+    """
+    crosswalk_match_exclude = {"CAMD Unmatched", "Manual CAMD Excluded"}  # noqa: F841
+    data = (
+        pd.read_csv(path)
+        .query(
+            "MATCH_TYPE_GEN not in @crosswalk_match_exclude and CAMD_STATUS != 'RET'"
+        )
+        .astype(
+            {
+                "MOD_EIA_PLANT_ID": int,
+                "MOD_CAMD_UNIT_ID": "string",
+                "MOD_CAMD_GENERATOR_ID": "string",
+                "MOD_EIA_GENERATOR_ID_GEN": "string",
```

Comment:
Just curious, `str` won't work here?

Reply:
`str` results in a columns of type `object` (older-style behavior), while `"string"` results in a `StringArray` (newer-style). I think the newer is preferred since it is more explicitly text data, while `object` was the previous catch-all.

https://pandas.pydata.org/pandas-docs/stable/user_guide/text.html

Reply:
Aha, good to know.